src/Microsoft.AspNetCore.Mvc.Core/TypeFilterAttribute.cs

```diff
@@ -9,12 +9,20 @@
 
 namespace Microsoft.AspNetCore.Mvc
 {
+    /// <summary>
+    /// A filter that activates another filter of type <see cref="ImplementationType"/>.
+    /// </summary>
```

Comment:
@rynowak we use "activates" in a few other places e.g. `Controller.ViewData` is `activated when <see cref="Controllers.IControllerActivator"/> activates`. Do those other uses (at least those outside `IRazorPageActivator` and similar) need more words? What words are you suggesting?

Without a specific suggestion, I'll switch to "creates" here.